docs/api/web-contents.md

```diff
@@ -221,6 +221,20 @@ Emitted when `webContents` wants to do basic auth.
 
 The usage is the same with [the `login` event of `app`](app.md#event-login).
 
+### Event: 'find-in-page-response'
+
+Returns:
+
+* `event` Event
+* `result` Object
+  * `requestId` Integer
+  * `matches` Integer __Optional__ - Number of Matches.
+  * `selectionArea` Object __Optional__ - Coordinates of first match region.
+  * `finalUpdate` Boolean - Indicates if more responses are to follow.
```

Comment:
The `finalUpdate` should be put bellow the `requestId`, so the required parameters are grouped together.